NEXT_CHANGELOG.md

```diff
@@ -28,11 +28,11 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 ## ❗ BREAKING ❗
 ## 🚀 Features
 
-### Add support for dhat based heap profiling [PR #XXXX](https://github.com/apollographql/router/pull/XXXX))
+### Add support for dhat based heap profiling [PR #1829](https://github.com/apollographql/router/pull/1829))
```

Comment:
this is the wrong changelog?

Reply:
Yeah, I was just being lazy and fixing it here rather than raising a new PR...

Reply:
I wasn't going to put in a changelog entry for this change. It doesn't feel big enough to warrant mentioning. I can add one if you think it needs one.

Reply:
I think there should be one yes. It can be put in the maintenance section, it's not a functional change

Reply:
Done: https://github.com/apollographql/router/pull/2030/commits/2817f79e3851a6a7c24db8e56e62390474c61d78